src/components/calcite-color-hex-input/calcite-color-hex-input.tsx

```diff
@@ -218,4 +228,10 @@ export class CalciteColorHexInput {
   private formatForInternalInput(hex: string): string {
     return hex.replace("#", "");
   }
+
+  private nudgeRGBChannels(color: Color, amount: number): Color {
```

Comment:
Another option would be to use map rather than restructuring and applying to channels individually. 

```js
const channels = color.array().map((channel) => channel + amount);
return Color.rgb(...channels);
```

Reply:
💥